migrations/README

```diff
@@ -0,0 +1,4 @@
+This is a database migration repository.
```

Comment:
Can you move these docs to the parent README?

Can you try to add the commands that you normally use to make it work?